app/script/view_model/panel/ConversationDetailsViewModel.js

```diff
@@ -76,6 +84,15 @@ z.viewModel.panel.ConversationDetailsViewModel = class ConversationDetailsViewMo
             }
             this.userParticipants.push(userEntity);
           });
+
+        const participantCount = this.userParticipants().length;
+        if (participantCount > ConversationDetailsViewModel.CONFIG.MAX_USERS_VISIBLE) {
+          const removeCount = participantCount - ConversationDetailsViewModel.CONFIG.REDUCED_USERS_COUNT;
+          this.userParticipants.splice(5, removeCount);
```

Comment:
No need to calculate the delete count. Omitting this in the splice will remove everything from your start value.